RevenueCat/Scripts/Purchases.cs

```diff
@@ -927,27 +1016,44 @@ public class EntitlementInfoResponse
         public bool isActive;
         public bool willRenew;
         public string periodType;
-        public long latestPurchaseDate;
-        public long originalPurchaseDate;
-        [CanBeNull] public long expirationDate;
+        public long latestPurchaseDateMillis;
+        public long originalPurchaseDateMillis;
+        [CanBeNull] public long expirationDateMillis;
         public string store;
         public string productIdentifier;
         public bool isSandbox;
-        [CanBeNull] public long unsubscribeDetectedAt;
-        [CanBeNull] public long billingIssueDetectedAt;
+        [CanBeNull] public long unsubscribeDetectedAtMillis;
+        [CanBeNull] public long billingIssueDetectedAtMillis;
     }
-    
+
     [Serializable]
     public class IntroEligibilityResponse
     {
         public int status;
         public string description;
     }
-    
+
     [Serializable]
     public class MapResponse<K, V>
     {
         public List<K> keys;
         public List<V> values;
     }
+
+    [Serializable]
+    public class ProductResponse
+    {
+        public string title;
+        public string identifier;
+        public string description;
+        public float price;
+        public string price_string;
+        [CanBeNull] public string currency_code;
+        public float intro_price;
+        public string intro_price_string;
+        public string intro_price_period;
+        public string intro_price_period_unit;
+        [CanBeNull] public int intro_price_period_number_of_units;
+        [CanBeNull] public int intro_price_cycles;
+    }
```

Comment:
When this PR is merged, I would like to move all these classes to their own class.

Reply:
+100